lib/application.js

```diff
@@ -72,6 +73,18 @@ var Application = module.exports = function(settings) {
     cookieParser()
   ];
 
+  // Settings cors middleware.
+  if (this.settings && this.settings.cors) {
+    var options = {};
+
+    if (typeof this.settings.cors === 'boolean') {
```

Comment:
Let's make `this.settings.cors` always be an object to avoid confusion. So this would be `if (this.settings.cors && this.settings.cors.enabled)` then always pass in the whole `this.settings.cors` to the middleware if there's no problem with it.